index.html

```diff
@@ -382,56 +379,84 @@ <h3 id="abtMeText" class="abtMeText">
     </div>
   </section>
   <section id="contact">
-    <div id="contactMeTextPH">
       <h2 id="contactMeText">
-        I'm always<br>interested in<br>
-        hearing about<br>new projects, so<br>
-        if you'd like to<br>chat please get<br>
-        in touch.
+        I'm always interested in 
+        hearing about new projects, so if you'd like to 
+        chat please get in touch.
       </h2>
-    </div>
     <div id="contactMeBlankSpace"></div>
-    <form id="contactForm" action="https://formspree.io/f/moqbvdgn" method="post">
-      <input type="text" name="name" id="nameFormBox" class="formBox1" placeholder="Full Name" maxlength="30" required>
-      <input type="email" name="email" id="mailFormBox" class="formBox1" placeholder="Email Adress" required>
-      <textarea name="comments" id="commentsFormBox" class="formBox2"
-        maxlength="500" required>Any comments?</textarea>
-      <button type="submit" id="submitFormBttn">Get in touch</button>
-    </form>
-    <div id="smI">
-      <ul id="socialMediaIcons2">
-        <li>
-          <a href="https://twitter.com/ricmtza">
-            <img class="socialMediaIcons" src="Images/twitter.png" alt="Twitter">
-          </a>
-        </li>
-        <li>
-          <a href="http://www.linkedin.com/in/ricardomtz7714">
-            <img class="socialMediaIcons" src="Images/Linkedin.png" alt="Linkedin">
-          </a>
-        </li>
-        <li>
-          <a href="https://medium.com/">
-            <img class="socialMediaIcons" src="Images/medium.png" alt="Medium">
-          </a>
-        </li>
-        <li>
-          <a href="https://github.com/rmrmrmr">
-            <img class="socialMediaIcons" src="Images/github.png" alt="GitHub">
-          </a>
-        </li>
-        <li>
-          <a href="https://angel.co/">
-            <img class="socialMediaIcons" src="Images/angellist.png" alt="Angelist">
-          </a>
-        </li>
-      </ul>
-    </div>
-    <div id="contactEmailPH">
-      <p id="contactEmail">ricardomartineza77@gmail.com</p>
+    <div id="bottomBackgroundImage">
+      <form id="contactForm" action="https://formspree.io/f/moqbvdgn" method="post">
+        <input type="text" name="name" id="nameFormBox" class="formBox1" placeholder="Full Name" maxlength="30" required>
+        <input type="email" name="email" id="mailFormBox" class="formBox1" placeholder="Email Adress" required>

```

Comment:
Please make sure that your form matches the [FIGMA design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). The [FIGMA template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)'s form has `First name` and `Last name` input fields next to each other. You only have one input field `Full name`.

Your design:

![image](https://user-images.githubusercontent.com/38088818/187792723-c9d1b264-58ef-4c3f-ac47-1b81f9fbaff6.png)

Expected design:

![image](https://user-images.githubusercontent.com/38088818/187792919-ab2820a5-ca1e-414d-bc64-f6cfdc2bcf1c.png)

- Kindly fix the borders of your input fields as well. Yours are black when they should be grey as per design specifications.

![image](https://user-images.githubusercontent.com/38088818/187792234-7a3e8e6b-bbac-430d-b135-13ddae342b69.png)

Reply:
Hi @KDlamini ! Thanks for your comments. 🙌
As per the first one, it is specified on the project requirements (https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)  to ignore the Figma design on the two inputs issue and keep the input for the full name on every template version. 

I'll make sure to change the border colors on the next commit!

Reply:
@rmrmrmr okay no problem 👍🏾